pyproject.toml

```
@@ -1,58 +1,94 @@
-[tool.poetry]
+[build-system]
+requires = ["hatchling", "uv-dynamic-versioning"]
+build-backend = "hatchling.build"
+
+[project]
 name = "gokart"
-version = "0.0.0"  # using poetry-dynamic-versioning
 description="Gokart solves reproducibility, task dependencies, constraints of good code, and ease of use for Machine Learning Pipeline. [Documentation](https://gokart.readthedocs.io/en/latest/)"
-authors = ["M3, inc."]
+authors = [
+  {name = "M3, inc."}
+]
 license = "MIT"
 readme = "README.md"
-homepage = "https://github.com/m3dev/gokart"
-repository = "https://github.com/m3dev/gokart"
-documentation = "https://gokart.readthedocs.io/en/latest/"
+requires-python = ">=3.9, <4"
+dependencies = [
+  "luigi",
+  "boto3",
+  "slack-sdk>=3",
```

[nits]

https://github.com/m3dev/gokart/pull/429/files?diff=split&w=0#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L22

`>=3` is differennt from `"^3"`, now it allows 4.x and greater. 

I think it doesn't matter, so just

```suggestion
  "slack-sdk",
```

---

`backoff` too ?

---

backoff removed in [df69966](https://github.com/m3dev/gokart/pull/429/commits/df6996682e24e38cdd7f644da7a67856f7e9adb4)